app/src/main/java/com/geekskool/dhobi/Models/Course.java

```diff
@@ -0,0 +1,73 @@
+package com.geekskool.dhobi.Models;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+
+/**
+ * Created by manisharana on 16/2/17.
+ */
+public class Course implements Serializable{
+    private int duration;
+    private ArrayList<Student> students;
+    private String location;
+    private long startDate;
+    private long endDate;
+    private String desc;
+
+    public Course(int duration, String location, long startDate, long endDate, String desc) {
+        this.duration = duration;
+        this.location = location;
+        this.startDate = startDate;
+        this.endDate = endDate;
+        this.desc = desc;
+    }
+
+    public int getDuration() {
+        return duration;
+    }
+
+    public void setDuration(int duration) {
```

Comment:
Avoid "setters". Domain objects should be Value Objects, which are immutable once created. (Sorry, this is another topic Manisha and Priyanka missed.)